doc/release-notes.md

```diff
@@ -21,78 +21,661 @@ How to Upgrade
-We have removed all mining code from Windows and Mac binaries, which should avoid most of the false-positive alerts
-from anti virus software. Linux builds are not affected. The mining code found in `dash-qt` and `dashd` are only meant
-for regression/integration tests and devnets, so there is no harm in removing this code from non-linux builds.
-
-Fixed an issue with invalid merkle blocks causing SPV nodes to ban other nodes
-------------------------------------------------------------------------------
-A fix that was introduces in the last minor version caused creation of invalid merkle blocks, which in turn cause SPV
-nodes to ban 0.13.2 nodes. This can be observed on mobile clients which have troubles maintaining connections. This
-release fixes this issue and should allow SPV/mobile clients to sync with upgraded nodes.
-
-Hardened spork15 value to 1047200
----------------------------------
-We've hardened the spork15 block height to 1047200, which makes sure that syncing from scratch will always work, no
-matter if spork15 is received in-time or not.
-
-Bug fixes/Other improvements
-----------------------------
-There are few bug fixes in this release:
-- Fixed an issue with transaction sometimes not being fully zapped when `-zapwallettxes` is used
-- Fixed an issue with the `protx revoke` RPC and REASON_CHANGE_OF_KEYS
-
- 0.13.3.0 Change log
+DIP0004 - Coinbase Payload v2
+-----------------------------
+Coinbase Payload v2 introduces new field `merkleRootQuorums` which represents the merkle root of
+all the hashes of the final quorum commitments of all active LLMQ sets. This allows SPV clients
+to verify active LLMQ sets and use this information to further verify ChainLocks and LLMQ-based
+InstantSend messages.
+
+https://github.com/dashpay/dips/blob/master/dip-0004.md#calculating-the-merkle-root-of-the-active-llmqs
+
+DIP0008 - ChainLocks
+--------------------
+This version introduces ChainLocks, a technology for near-instant confirmation of blocks and
+finding near-instant consensus on the longest valid/accepted chain. ChainLocks leverages LLMQ
+Signing Requests/Sessions to accomplish this. ChainLocks activation relies on DIP0004 activation and
+`SPORK_19_CHAINLOCKS_ENABLED` spork.
+
+Read more: https://github.com/dashpay/dips/blob/master/dip-0008.md
+
+DIP000? - LLMQ-based InstantSend
+--------------------------------
+Read more: https://github.com/dashpay/dips/blob/master/dip-000?.md
+
+Network
+------
+Legacy messages `mnw`, `mnwb`, `mnget`, `mnb`, `mnp`, `dseg`, `mnv`, `qdcommit` and their corresponding
+inventory types (7, 10, 14, 15, 19, 22) are no longer suported.
+
+Message `version` is extended with a 256 bit field - a challange sent to a masternode. Masternode which
+received such a challange must reply with new p2p message `mnauth` directly after `verack`. This `mnauth`
+message must include a signed challenge that was previously sent via `version`.
+
+Read more: https://github.com/dashpay/dips/blob/master/dip-000?.md
```

Comment:
Is this worth a DIP?

Reply:
:thinking: It looks pretty minor to me. What would a DIP cover that isn't here?

Reply:
Yeah, I dropped this line 😊